File: Beginner/1064 (Positives and Average)/1064.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int acm = 0;
            double aux =  0;
            for (int i = 0; i < 6; i++)
            {
                double x = double.Parse(Console.ReadLine());
                if (x > 0)
                {
                    acm++;
                    aux += x;
                }
            }

      
            Console.WriteLine(acm + " valores positivos");
            Console.WriteLine(string.Format("{0:0.0}", aux/acm)) ;
        }
    }
}

File: Beginner/1064 (Positives and Average)/1064.py
acmP = int()
aux = float()

for i in range (6):
    x = float(input())
    if(x > 0):
        acmP += 1
        aux = aux + x


print(acmP, "valores positivos")
print("{:0.1f}".format(aux/acmP))